File: CalculatorExercise/Operation.js
class Operation {

	calculate(numbers, operator) {
		switch(operator) {
		case '+':
			return numbers[0] + numbers[1];
		case '-':
			return numbers[0] - numbers[1];
		case '/':
			return numbers[0] / numbers[1];
		case 'x':
			return numbers[0] * numbers[1];			
		}
	}
}

module.exports = Operation

File: lucasHenrique_Desafio4/src/br/com/web/model/Operation.java
package br.com.web.model;

public enum Operation {
	
	SUM('+'), SUBTRACT('-'), MULTIPLY('*'), DIVIDE('/');
	
	private final char operator;
	
	Operation(char operator) {
		this.operator = operator;
	}
	
	public static Operation valueOf(char operator) {
			
		if (operator == SUM.operator) {
			return SUM;
		} 
		
		if (operator == DIVIDE.operator) {
			return DIVIDE;
		}
		
		if (operator == MULTIPLY.operator) {
			return MULTIPLY;
		}
		
		return SUBTRACT;
	}
	
	
	
	
}
